### PR TITLE
resource/alicloud_cen_transit_router_vpc_attachment: Added retry strategy for error code IncorrectStatus.VpcSwitch; Removed the field route_table_association_enabled, route_table_propagation_enabled

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

File: alicloud/resource_alicloud_cen_transit_router_vpc_attachment.go [hunk 1]
```diff
@@ -113,11 +103,21 @@ func resourceAlicloudCenTransitRouterVpcAttachment() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"route_table_association_enabled": {
```
Comment (MrWolong) [inline alicloud/resource_alicloud_cen_transit_router_vpc_attachment.go:106]: route_table_association_enabled和route_table_propagation_enabled已从接口文档中删除

File: alicloud/service_alicloud_cbn.go [hunk 2]
```diff
@@ -922,53 +922,77 @@ func (s *CbnService) CenTransitRouterVbrAttachmentStateRefreshFunc(id string, fa
...
+			}
+		}
+
+		if nextToken, ok := response["NextToken"].(string); ok && nextToken != "" {
+			request["NextToken"] = nextToken
+		} else {
+			break
 		}
 	}
-	object = v.([]interface{})[0].(map[string]interface{})
```
Comment (MrWolong) [inline alicloud/service_alicloud_cbn.go:971]: 这种查询逻辑是错误的，不应该默认取第一个值

File: website/docs/r/cen_transit_router_vpc_attachment.html.markdown [hunk 3]
```diff
@@ -102,9 +109,10 @@ The `zone_mappings` supports the following:
 
 The following attributes are exported:
 
-* `id` - ID of the resource, It is formatted to `<transit_router_id>:<transit_router_attachment_id>`.
```
Comment (MrWolong) [inline website/docs/r/cen_transit_router_vpc_attachment.html.markdown:105]: 资源id是错的，代码中的id拼接是cenId和transitRouterAttachmentId

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**